### PR TITLE
Update 01-start-here.md

### PR DESCRIPTION
**Pull request by jianreis**
First of the new Docs 2.0 propose edits to align Snaplet docs to the new onboarding flow and various other changes we've made to the product.

### CODE REVIEW

File: docs/03-getting-started/01-start-here.md [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
+
+Once you're comfortable with the basics of how Snaplet works, you can connect Snaplet to your own database, explore the more powerful CLI, or even [self-host](/guides/self-hosting) Snaplet in your own trusted infrastructure.
+
+Let's dive in!
 
 <div style={{textAlign: 'center'}}>
 
 ![Give me rubs... So that I can scratch you!](/img/snaplet-playing.svg)
 
-</div>
```
Comment (peterp) [inline docs/03-getting-started/01-start-here.md:14]: I think removing this is the mistake.

Comment (jianreis) [reply docs/03-getting-started/01-start-here.md:14]: > I fix.

File: docs/03-getting-started/01-start-here.md [hunk 2]
```diff
@@ -1,32 +1,49 @@
-# Before we begin...
+# Start here
+Welcome to Snaplet! To help you get started quickly, we'll guide you through a sample workflow using the Snaplet Web App (what we call Snaplet Cloud) and the Snaplet CLI. Using these two tools will allow you to capture, transform, and restore a snapshot of your source database into a target development database.
```
Comment (peterp) [inline docs/03-getting-started/01-start-here.md]:
```suggestion
Welcome to Snaplet! To help you get started quickly, we'll guide you through a sample workflow using the Snaplet Cloud and the Snaplet CLI. Using these two tools will allow you to capture, transform, and restore a snapshot of your source database into a target development database.
```

File: docs/03-getting-started/01-start-here.md [hunk 3]
```diff
@@ -1,32 +1,49 @@
...
+Once you're comfortable with the basics of how Snaplet works, you can connect Snaplet to your own database, explore the more powerful CLI, or even [self-host](/guides/self-hosting) Snaplet in your own trusted infrastructure.
+
+Let's dive in!
 
 <div style={{textAlign: 'center'}}>
 
 ![Give me rubs... So that I can scratch you!](/img/snaplet-playing.svg)
-
 </div>
+## Getting Started with Snaplet Cloud
```
Comment (peterp) [inline docs/03-getting-started/01-start-here.md:18]:
```suggestion

## Getting Started with Snaplet Cloud
```

**Review by peterp — APPROVED** [review]

**Pull request merged**